frontend/pages/admin/IntegrationsPage/IntegrationNavItems.tsx

```diff
@@ -4,10 +4,9 @@ import { ISideNavItem } from "../components/SideNav/SideNav";
 import Integrations from "./cards/Integrations";
 import Mdm from "./cards/MdmSettings/MdmSettings";
 import AutomaticEnrollment from "./cards/AutomaticEnrollment/AutomaticEnrollment";
+import Calendars from "./cards/Calendars/Calendars";
 
-const getFilteredIntegrationSettingsNavItems = (
-  isSandboxMode = false
-): ISideNavItem<any>[] => {
+const getFilteredIntegrationSettingsNavItems = (): ISideNavItem<any>[] => {
```

Comment:
since this is no longer doing any filtering, it can just be a constant instead of a function